cpp/src/cider/common/Arena.h

```diff
@@ -22,11 +22,25 @@
 
 #pragma once
 
+#include <../include/cider/CiderAllocator.h>
```

Comment:
change this include path with unified style

Reply:
done